tests/test_stat_result.py

```diff
@@ -12,6 +12,9 @@ def setUp(self) -> None:
         self.stat_result = StatResult("Test X", "X", "Stat", 1, 0.02,
```

Comment:
Maybe we could refactor this one's name too?
```suggestion
        self.stat_result_significant = StatResult("Test X", "X", "Stat", 1, 0.02,
```

Reply:
Yes, I'll add then when I go over the code again as it requires a few additional changes